configs/prettier.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-PRETTIER="`npm bin`/prettier --no-semi --no-bracket-spacing --print-width 120 --single-quote --trailing-comma es5"
+PRETTIER="`npm bin`/prettier"
```

Comment:
We can use `npx prettier` here now.

Reply:
Yeah, originally I've created a shell script because didn't find a better way to safe the config. Perhaps we can get rid of it now.

Reply:
Oh! Yeah, we could definitely move this into `pkg.json#scripts` entirely.

Reply:
I've done that now. (Since it's just in `pkg.json#scripts` now, there's no need for npx since the scripts are run with the npm bin directory in path.)